installer/Migration/V4_3_3/Migration.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace OrangeHRM\Installer\Migration\V4_3_3;
+
+use Doctrine\DBAL\Types\Types;
+
+class Migration extends \OrangeHRM\Installer\Util\V1\AbstractMigration
+{
+    /**
+     * @inheritDoc
+     */
+    public function up(): void
+    {
+        $this->getSchemaHelper()->changeColumn('hs_hr_employee', 'emp_number', [
+            'Type'=>Types::INTEGER,'Length'=> 7, 'Notnull' => true,'Autoincrement'=>true
```

Comment:
```suggestion
            'Type' => Types::INTEGER, 'Length' => 7, 'Notnull' => true, 'Autoincrement' => true
```
format the code using IDE

Reply:
FORMATTED